functors_task_1/student/include/impl.h

```diff
@@ -16,16 +29,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(const BeerBrand beer)
 {
+    return (beer != BeerBrand::None) && (beer < BeerBrand::Carlsberg);
```

Comment:
there is only 2 possible values, please use them to make this expression more explicit